harmoni_actuators/harmoni_tts/setup.py

```diff
@@ -6,7 +6,7 @@
 # fetch values from package.xml
 setup_args = generate_distutils_setup(
     packages=['harmoni_tts'],
-    package_dir={'': 'src'},
+    package_dir={'': 'nodes'},
```

Comment:
You can just remove these lines unless you plan to make a lib version that gets imported into the node. Typically nodes are not included in setup as we do not import nodes into other nodes, and therefore they do not need to be installed like libraries do.